src/conf_mode/bcast_relay.py

```diff
@@ -52,16 +54,14 @@ def verify(relay):
         if 'port' not in config:
             raise ConfigError(f'Port number mandatory for udp broadcast relay "{instance}"')
```

Comment:
I think we should not omit prepositions, and protocol names should also be capitalzied: `Port number is mandatory for UDP broadcast relay`.

Reply:
Done